relay-general/src/store/transactions/processor.rs

```diff
@@ -487,6 +493,12 @@ fn scrub_span_description(span: &mut Span) -> Result<(), ProcessingAction> {
         }
     }
 
+    if let Some(is_cache_like) = span.op.value().map(|op| op.starts_with("cache")) {
+        if is_cache_like && scrub_cache_keys(&mut scrubbed)? {
+            did_scrub = true;
+        }
+    }
```

Comment:
This check and the one above should be `else if`: No need to check the prefix again if it already matched another one. You could also simplify to something like

```suggestion
    if span.op.value().map_or(false, |op| op.starts_with("cache")) {
        did_scrub |= scrub_cache_keys(&mut scrubbed)?;
    }
```

Reply:
Address this in [cfb8dbc](https://github.com/getsentry/relay/pull/2129/commits/cfb8dbcc4ecffd755b080b8bf484b4372af0cf3a) instead of this PR to avoid more merge conflicts.